### ISSUE TITLE
class PickleShareDB still inherits from collections instead of collections.abc

### ISSUE THREAD

**Issue by jmfrank63** [pickleshare/pickleshare#27]
deprecation warning on python 3.7:

```
... 
from pickleshare import PickleShareDB
../../../../virtualenv/python3.7.0/lib/python3.7/site-packages/pickleshare.py:66: in <module>
    class PickleShareDB(collections.MutableMapping):
../../../../virtualenv/python3.7.0/lib/python3.7/collections/__init__.py:52: in __getattr__
    DeprecationWarning, stacklevel=2)
E   DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
```
Several dependencies including IPython 6.5.0 fail on travis CI tests now.

**Comment by takluyver** [comment]
Thanks, want to make a PR?

**Comment by jmfrank63** [comment]
Hi Thomas, yes, give me a few hours once I get home tonight.
Johannes

**Comment by takluyver** [comment]
Great, thanks. :-)

**Comment by pquentin** [comment]
(To be fair, it only fails for trio because `-Werror` is used.)

**Comment by takluyver** [comment]
I'm glad someone was using that option - catching it now means we can put out a fix calmly instead of scrambling to fix a real bug when 3.8 is out.

**Comment by takluyver** [comment]
Closed by #28.
